README.md

```diff
@@ -38,7 +38,7 @@ final class AppDelegate: ApplicationDelegate {
 ## Getting started with **AppKid** in your project
 **AppKid** depends on mulitple opensource projects. Below are the instructrions on how to set those up for Debian-based Linux distributions. RPM-Based instructions will be added some time later.
 
-- <details>
+- <details open>
```

Comment:
These walls of text are wrapped with \<details\> tag for a reason. If you prefer to have them open it's better to move the out from README.md file to a new file Installation.md under Documentation directory. The reason for this is README file is the first thing you see when you open the repo. If it's a giant wall of text immediately humans tend to lose interest very fast due to immediate fatigue induced by it

Reply:
closed them as u requested, but added text to help users know the instructions are hidden behind a clickable thing.